### PR TITLE
custom rpc responder

### PR DESCRIPTION
**Pull request by fjsousa**
- Enables passing a custom rpc-responder
- solves https://github.com/originrose/legend-live/issues/1691 (`false` return values get converted in `nil`)

### CODE REVIEW

File: scripts/core-access [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-export VAULT_ADDR='https://thinktopic.com:8200'
+export VAULT_ADDR='https://vault.originrose.com:8200'
```
Comment (djzurawski) [inline scripts/core-access:2]: Ahmed mentioned removing this and passing it in travis settings. We're working on fixing the build now

Comment (AHAAAAAAA) [reply scripts/core-access:2]: It's in the settings and can be safely removed. The build is failing due to compilation issues now.

**Comment by fjsousa** [comment]
tests passed

**Comment by fjsousa** [comment]
one of the job passes with 
```
lein test test-api

Ran 4 tests containing 25 assertions.

0 failures, 0 errors.

The command "lein test" exited with 0.

Done. Your build exited with 0.`


but the other hangs after running the tests. the tests pass and the traces that get logged come from [here](https://github.com/originrose/peer/blob/rpc-error-responder/test/test_api.clj#L25)

**Comment by fjsousa** [comment]
tests timing out in the PR job, no idea why

```
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.

Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received

The build has been terminated
```

**Comment by fjsousa** [comment]
anyone fixed the PR job?

**Comment by fjsousa** [comment]
can I merge?

**Pull request merged**